<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultData</class>
 <widget class="QDialog" name="ResultData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>1366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ResultData</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QCheckBox" name="cB_sysLog">
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>系统日志</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bG_checkBox</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="cB_operationLog">
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>操作日志</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bG_checkBox</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="cB_alarmLog">
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>报警日志</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bG_checkBox</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="cB_data">
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>检测数据</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bG_checkBox</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cB_model">
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>模板</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">bG_checkBox</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>筛选批号：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="MyLineEdit" name="lE_productionNum">
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>全部</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>筛选用户：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cB_LeadOut">
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>所有</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>筛选日期：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QDateEdit" name="dateEdit">
              <property name="font">
               <font>
                <family>幼圆</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="displayFormat">
               <string>yyyy/MM/dd</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>10</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>幼圆</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit_2">
              <property name="font">
               <font>
                <family>幼圆</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="displayFormat">
               <string>yyyy/MM/dd</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>eg: D:/myProjects/myFiles</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pB_Choose">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>选择目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_LeadOut">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_LeadOutClose">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 170, 127);</string>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bG_checkBox">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
